parcels/tools/timer.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+import time
+
+
+class Timer():
+    def __init__(self, name, parent=None, start=True):
+        self._start = -1
+        self._end = -1
```

Comment:
More pythonic to initialise these to None? And then later on do e.g. the oneliner:
```
def local_time(self):
    return self._t + time.time() - self._start if self._start else self._t
```